<ui version="4.0" >
 <class>TableWindow</class>
 <widget class="QMainWindow" name="TableWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>550</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>550</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="windowIcon" >
   <iconset resource="sfunctioner.qrc" >
    <normaloff>:/icons/resources/s.ico</normaloff>:/icons/resources/s.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QPushButton" name="pushButtonNext" >
    <property name="geometry" >
     <rect>
      <x>440</x>
      <y>290</y>
      <width>96</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;Next ></string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxGroup" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>10</y>
      <width>521</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
    <widget class="QTableWidget" name="tableWidgetTable" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>30</y>
       <width>411</width>
       <height>241</height>
      </rect>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode" >
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle" >
      <enum>Qt::DashDotDotLine</enum>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="rowCount" >
      <number>0</number>
     </property>
     <property name="columnCount" >
      <number>5</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QPushButton" name="pushButtonAdd" >
     <property name="geometry" >
      <rect>
       <x>440</x>
       <y>30</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text" >
      <string>&amp;Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemove" >
     <property name="geometry" >
      <rect>
       <x>440</x>
       <y>90</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text" >
      <string>&amp;Remove</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUp" >
     <property name="geometry" >
      <rect>
       <x>440</x>
       <y>130</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text" >
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDown" >
     <property name="geometry" >
      <rect>
       <x>440</x>
       <y>160</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text" >
      <string>Down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEdit" >
     <property name="geometry" >
      <rect>
       <x>440</x>
       <y>60</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text" >
      <string>&amp;Edit</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonBack" >
    <property name="geometry" >
     <rect>
      <x>340</x>
      <y>290</y>
      <width>96</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt; &amp;Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAbout_Qt" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuMenu" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="sfunctioner.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>TableWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>274</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
